### ISSUE TITLE
Multi accs support

### ISSUE THREAD

**Issue by Albertmoiseev** [InstaPy/InstaPy#165]
Dear contributors, 

I think that multi accounts support will be extremely useful.

Nominally, in the current version, one could create many instances. 
However, it is not very  convenient and it will be single-threading.

What do you think about this feature in general and the best way to implement it?

**Comment by eduardocalazansjr** [comment]
Selenium can't work with multi-thread, since is an browser automation tool.
The only way is running multiple accounts via cronjob, and it works like a charm.

Also, already discussed in another issue.

Also: running multi accounts in the same time on same ip will lead you to a soft-ban, maybe a perma-ban after that.

**Comment by timgrossmann** [comment]
closed because of the by @vlntdds mentioned issues...

Also... running multiple chrome browsers in parallel actually costs a whole lot of RAM
